modules/getting-started/p_next-steps.adoc

```diff
@@ -0,0 +1,19 @@
+// Metadata created by nebel
+//
+// UserStory: As an RH SSO customer, I want to now use the actual product.
+
+[id="next-steps"]
+= Choosing the next step
```

Comment:
Not sure this ads anything not covered where the production version is mentioned in Chapter 1. I'd probably just move the links to there and remove this chapter.

Reply:
Thanks. I am getting different opinions on this, but you could be right.

I addressed the other feedback for which I did not leave a comment also. Thanks again.